jsonrpc/src/methods/history.json

```diff
@@ -149,7 +154,12 @@
     ],
     "result": {
       "$ref": "#/components/contentDescriptors/TraceRecursiveFindContentResult"
-    }
+    },
+    "errors": [
```

Comment:
As I was making these adjustments in fluffy, I realized that we shouldn't return an error for the trace version of the call. Else we will not receive the actual trace which still contains useful information of the route the requests took.

So I think we should instead make the `content` and `utpTransfer` not required or perhaps even incorporate them in the `traceResultObject`. In the fluffy implementation it seems that the content field is already optional (cc @mynameisdaniil).

Reply:
I was thinking that in the trace versions of these calls, you would still return the trace but in the `"data"` field of the error